how-to-build.md

```diff
@@ -46,110 +42,64 @@ On OS X, Clang is used as the C/C++ compiler by default. Clang is installed
 as part of Xcode. Xcode 7.0 or newer is required, and can be installed via
 the Mac App Store.
 
-Configure, build & test
--------------------------
+## Configure, build & test
 
 Run the following commands to configure, build and test core:
 
-    sh build.sh config
-    sh build.sh build
-    sh build.sh test
-
-
-Building for Android
---------------------
-
-Building for Android required the NDK of version >= r10d installed.
-
-It can be built using the following command:
-
-    sh build.sh build-android
-
-Building for iOS, watchOS and tvOS
-----------------------------------
-
-Realm core may be built for iOS, watchOS and tvOS from an OS X machine.
-See the OS X section of [Prerequisites](#prerequisites) above for information
-about the version of Xcode that is required.
-
-To build for iOS:
-
-    sh build.sh build-ios
-
-This produces the following files and directories:
+    mkdir build-dir # create a build folder
+    cd build-dir
+    cmake ..
+    cmake --build
+    ctest
 
-    ios-lib/include/
-    ios-lib/librealm-ios.a
-    ios-lib/librealm-ios-dbg.a
-    ios-lib/realm-config
-    ios-lib/realm-config-dbg
+## Building for Android, iOS, watchOS and tvOS
 
-The `include` directory holds a copy of the header files, which are
-identical to the ones installed by `sh build.sh install`. There are
-two versions of the static library, one that is compiled with
-optimization, and one that is compiled for debugging. Each one
-contains code compiled for both iOS devices and the iOS simulator.
-Each one also comes with a `config` program that can be used to
-enquire about required compiler and linker flags.
+Building for Android required the NDK r10e installed.
 
-To build for watchOS:
+These targets can be built using the build.sh command:
 
-    sh build.sh build-watchos
+    ./build.sh -o android -a armeabi-v7a -t Release -v <X.Y.Z>
 
-The output is placed in watchos-lib, and is structured the same as
-is described for iOS above.
+The command shows the available options simply with:
 
-To build for tvOS:
+    ./build.sh
 
-    sh build.sh build-tvos
+These commands produce a tarball containing the realm static library
+and its include files.
```

Comment:
What's the new equivalent to what `sh build.sh build-cocoa` did in the old build system? `build.sh`'s help only mentions building for a specific platform and configuration, not creating the aggregate tarball that `build-cocoa` created and that the Cocoa binding expects.